tests/linkediting.js

```diff
@@ -19,22 +21,34 @@ import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
 /* global document */
 
 describe( 'LinkEditing', () => {
-	let editor, model, view;
+	let element, editor, model, view;
 
 	beforeEach( () => {
-		return VirtualTestEditor
-			.create( {
-				plugins: [ Paragraph, LinkEditing, Enter ]
+		element = document.createElement( 'div' );
+		document.body.appendChild( element );
+
+		return ClassicTestEditor
+			.create( element, {
+				plugins: [ Clipboard, Paragraph, LinkEditing, Enter ]
 			} )
 			.then( newEditor => {
 				editor = newEditor;
+
+				editor.model.schema.extend( '$text', { allowAttributes: 'bold' } );
+
+				editor.conversion.attributeToElement( {
+					model: 'bold',
+					view: 'b'
+				} );
+
 				model = editor.model;
 				view = editor.editing.view;
 			} );
 	} );
 
 	afterEach( () => {
 		editor.destroy();
```

Comment:
Would be safer to wait for a promise to resolve.

I'm using `async/await` in test, but can be done with removing an element and then returning the `editor.destroy()` promise.